notebooks/s3d-mil-nce-text-to-video-retrieval/s3d-mil-nce-text-to-video-retrieval.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    %pip install -q openvino-tokenizers --upgrade --pre --extra-index-url "https://storage.openvinotoolkit.org/simple/wheels/nightly"
</pre><p>it is better to install tokenizers and nightly in one line, because tokenizers depends on openvino and it may lead to installing openvino twice if they are not compatible (openvino-nightly package is also located in <a href="https://storage.openvinotoolkit.org/simple/wheels/nightly%22" target="_blank">https://storage.openvinotoolkit.org/simple/wheels/nightly</a>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Fixed</p>